package.json

```diff
@@ -43,21 +43,21 @@
     "react-dom": ">=16.8"
   },
   "dependencies": {
-    "@blueprintjs/core": "^3.25.0",
-    "@blueprintjs/icons": "^3.15.0",
-    "@blueprintjs/select": "^3.12.1",
-    "@fortawesome/fontawesome-common-types": "^0.2.28",
+    "@blueprintjs/core": "^3.29.0",
```

Comment:
Nice, QA will like this one: https://github.com/palantir/blueprint/pull/4180

Reply:
exactly :)